src/local-store.ts

```
@@ -106,3 +109,33 @@ export default class Auth {
+export class OfflineStore {
+  private static _instance: OfflineStore;
+
+  constructor(private secretStorage: SecretStorage) { }
+
+  static init(context: ExtensionContext): OfflineStore {
+    OfflineStore._instance = new OfflineStore(context.secrets);
+
+    return OfflineStore._instance;
+  }
+
+  async getViewOptions(): Promise<ViewOptionsState | {}> {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)